Adamant/Modules/Wallets/SecretWalletsService/SecretWalletsService.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  SecretWalletsService.swift
+//  Adamant
+//
+//  Created by Dmitrij Meidus on 22.01.25.
+//  Copyright © 2025 Adamant. All rights reserved.
+//
+
+protocol SecretWalletsService {
+    var publicWallet: PublicWalletServiceCompose { get }
+    var secretWallets: [String: SecretWalletServiceCompose] { get }
+    var currentWallet: PublicWalletServiceCompose { get }
```

Comment:
what's the difference between publicWallet and currentWallet? in init below seems like they are the same